packages/default/scss/notification/_layout.scss

```diff
@@ -71,15 +71,15 @@
 
         .k-notification-wrap {
 
-            > .k-icon {
+            > .k-notification-status,
+            > .k-notification-close {
                 margin-right: 0;
-                margin-left: $notification-icon-spacing;
+                margin-left: $kendo-notification-icon-spacing;
             }
 
-            > .k-i-close,
-            > .k-i-x {
+            > .k-notification-close {
```

Comment:
time and time, we've seen that we may need additional icons / actions. can you consider this to be inside a container?